src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPFactorSynchronisationService.kt

```diff
@@ -89,11 +90,6 @@ class CSIPFactorSynchronisationService(
         "nomisCSIPFactorId" to event.csipFactorId,
       )
 
-    if (event.auditModuleName == "DPS_SYNCHRONISATION") {
```

Comment:
Previous PR comments highlighted we won't know where the delete was coming from - so this check is irrelevant